code/model/UserDefinedForm.php

```diff
@@ -879,4 +726,44 @@ public function finished()
             'Form' => '',
         ));
     }
+
+	/**
+	 *
+	 * @param array $watch
```

Comment:
Please add a description for this. Also this method should either be protected so it can be overloaded by someone who wants to, or provide an extension point instead